index.html

```diff
@@ -1423,7 +1423,7 @@ <h3>Monitoring Status Lists</h3>
       </p>
 
       <p>
-[=Issuers=] can provide a level of reprieve from this privacy concern
+[=Issuers=] can provide a level of reprieve from this privacy concern for
 [=holders=] by revoking and reissuing effectively the same
 [=verifiable credential=] on a timeline that is relatively short in nature.
```

Comment:
Small language fix... Could get its own PR, but it seems harmless to add it here, where I spotted it.
```suggestion
[=verifiable credential=] on a relatively brief timeline.
```